qa/rpc-tests/mempool_reorg.py

```diff
@@ -54,9 +54,9 @@ def run_test(self):
         # and make sure the mempool code behaves correctly.
         b = [ self.nodes[0].getblockhash(n) for n in range(101, 105) ]
         coinbase_txids = [ self.nodes[0].getblock(h)['tx'][0] for h in b ]
-        spend_101_raw = self.create_tx(coinbase_txids[1], node1_address, 50)
-        spend_102_raw = self.create_tx(coinbase_txids[2], node0_address, 50)
-        spend_103_raw = self.create_tx(coinbase_txids[3], node0_address, 50)
```

Comment:
* By changing the amount you send in a raw transaction, you adjust the fee. Thus we get a `0.01 BTC` fee here. (The exact fee does not matter for this test but it should be `0.01 BTC <= fee <= 0.00001 BTC`)
* Current master is broken with free transactions, so we need the fee here